scripts/fastsp_script/fsp_test.py

```diff
@@ -2,39 +2,98 @@
 import sys
 
 """
-###remember sys.argv - can create variables using indexes with sys.argv!  
-    
+###remember sys.argv - can create variables using indexes with sys.argv!      
 """
 
 dir = "perturbed_alignments_Drosophila_AA/EMGT00050000000018.Drosophila.001_AA"
 slash = "/"
+fas = ".fasta"
 
-#print(sys.argv)
-
-outfile = output_fs.txt
-#will need to be .csv later 
+infile = open("output_fs.txt", "w+")
```

Comment:
This is pretty dangerous - don't keep a file ALWAYS OPEN when it's being constantly written to and overwritten. Open and close it each time you want to open it within the loop itself.